webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePattern.js

```diff
@@ -12,18 +22,22 @@ const EmptyStatePattern = props => {
     header,
     description,
   } = props;
+  const iconName = `${upperFirst(icon)}Icon`;
+  const iconClass = pficons[iconName];
```

Comment:
Awesome work, making this generic,
since it's a component can you rename it into upperCase?
```suggestion
  const Icon = pficons[iconName];
```